### PR TITLE
Php Clean code format PSR-12

### PR DESCRIPTION
**Pull request by abeka-buckaroo**

### CODE REVIEW

File: src/PaymentMethods/KlarnaPay/Models/Recipient.php [hunk 1]
```diff
@@ -20,9 +20,14 @@
 
 namespace Buckaroo\PaymentMethods\KlarnaPay\Models;
 
-use Buckaroo\Models\{Address, Company, Email, Person, Phone, ServiceParameter};
```
Comment (vegimcarkaxhija) [inline src/PaymentMethods/KlarnaPay/Models/Recipient.php:23]: for me this one is more readable

File: src/PaymentMethods/PaymentMethodFactory.php [hunk 2]
```diff
@@ -68,44 +68,61 @@ class PaymentMethodFactory
      * @var array|\string[][]
      */
     private static array $payments = [
-        ApplePay::class                 => ['applepay'],
```
Comment (vegimcarkaxhija) [inline src/PaymentMethods/PaymentMethodFactory.php:71]: for me this is more readable

File: src/Handlers/Reply/ReplyHandler.php [hunk 3]
```diff
@@ -140,11 +135,10 @@ public function isValid()
      */
     public function data($key = null)
     {
-        if($key)
-        {
+        if ($key) {
             return $this->data[$key] ?? $this->data[strtolower($key)] ?? $this->data[strtoupper($key)] ?? null;
```
Comment (Rinor12010) [inline src/Handlers/Reply/ReplyHandler.php:139]: When making comparisons of more than 2 variables break them to multiple lines

File: src/Models/ServiceList.php [hunk 4]
```diff
@@ -20,7 +20,9 @@
 
 namespace Buckaroo\Models;
```
Comment (Rinor12010) [inline src/Models/ServiceList.php:22]: The shorter version is better

File: src/PaymentMethods/Billink/Models/Recipient.php [hunk 5]
```diff
@@ -20,7 +20,12 @@
 
 namespace Buckaroo\PaymentMethods\Billink\Models;
```
Comment (Rinor12010) [inline src/PaymentMethods/Billink/Models/Recipient.php:22]: The shorter version is better

File: src/PaymentMethods/CreditManagement/Models/Invoice.php [hunk 6]
```diff
@@ -20,7 +20,13 @@
 
 namespace Buckaroo\PaymentMethods\CreditManagement\Models;
```
Comment (Rinor12010) [inline src/PaymentMethods/CreditManagement/Models/Invoice.php:22]: The shorter version is better

File: src/PaymentMethods/KlarnaPay/Models/Recipient.php [hunk 7]
```diff
@@ -20,9 +20,14 @@
 
 namespace Buckaroo\PaymentMethods\KlarnaPay\Models;
```
Comment (Rinor12010) [inline src/PaymentMethods/KlarnaPay/Models/Recipient.php:22]: The shorter version is better

File: src/PaymentMethods/PaymentMethodFactory.php [hunk 8]
```diff
@@ -68,44 +68,61 @@ class PaymentMethodFactory
...
+            'podiumcadeaukaart', 'shoesaccessories', 'webshopgiftcard',
+            'wijncadeau', 'wonenzo', 'yourgift',
+            'vvvgiftcard', 'parfumcadeaukaart'
+        ],
+        Trustly::class => ['trustly'],
+        BankTransfer::class => ['transfer'],
+        RequestToPay::class => ['requesttopay'],
+        WeChatPay::class => ['wechatpay'],
+        BuckarooVoucher::class => ['buckaroovoucher'],
     ];
```
Comment (Rinor12010) [inline src/PaymentMethods/PaymentMethodFactory.php:126]: The original format is better

File: src/PaymentMethods/Tinka/Models/Recipient.php [hunk 9]
```diff
@@ -20,8 +20,12 @@
 
 namespace Buckaroo\PaymentMethods\Tinka\Models;
```
Comment (Rinor12010) [inline src/PaymentMethods/Tinka/Models/Recipient.php:22]: The shorter version is better

**Review by harli91 — APPROVED** [review]

File: src/BuckarooClient.php [hunk 10]
```diff
@@ -118,8 +118,7 @@ public function client()
      */
     private function getConfig(string $websiteKey, string $secretKey, string $mode = null): ?Config
     {
-        if($websiteKey && $secretKey)
-        {
```
Comment (ShuCh3n) [inline src/BuckarooClient.php:122]: My personal preference is to have brackets on the next line. This way, it's easier to see where it starts.

File: src/Models/ServiceList.php [hunk 11]
```diff
@@ -20,7 +20,9 @@
...
 
-use Buckaroo\Services\ServiceListParameters\{DefaultParameters, ModelParameters, ServiceListParameter};
```
Comment (ShuCh3n) [inline src/Models/ServiceList.php:23]: A grouped declaration is easier to read.

**Review by ShuCh3n — CHANGES_REQUESTED** [review]

**Review by ShuCh3n — APPROVED** [review]

**Pull request merged**